sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,174 @@
+
+### Documentation
+- Update sycl_ext_oneapi_local_memory [457251feceb81845de962383f227e14522d4cf96]
+- Add get-coord API and general query example to `sycl_ext_intel_matric` [579f1dec0b5b2ecd177dfb95c3d1522f57079cc6] [38ac2124c1e8034c37056256f9864e475128ad49]
+- Update design doc for new offloading model [115808f5128f33321c68b9dead4fcd25f6a2ed49]
+- Update immediate command list usage [ad973f3383e265a1e209b61559e24cec37c5d111]
+- Add overloads with sycl::queue for some functions in `sycl_ext_oneapi_bindless_images` [4d0c552985bd0e4fc16740a0fc89393ef7fe3d04]
+- Add queue requirement when replay a graph [2c53815c6c1e3f8e40c5b6dbe3e5dc2aebd4fa37]
+
+## Bug Fixes
+### SYCL Compiler
+- Fix backend option passing issues with multiple targets [12ca2db0e30b155c7d6bf14320dd1ba468ac9294]
+- Fix -fgpu-rdc option for CUDA [f7595ac7527e67a59e33550851366db7d886ac3f]
+- Fix triple in offload mismatch warning [0fd9a4eb17a2db0fed151ba9d975804322966bcc]
+- Fix a bug when -enable-global-offset=false for AMDGU and NVPTX targets [00cf4c29740b2ec8d027e59e49e376a211599a58]
+
+### SYCL Library
+- Fix stack corruption in sycl::modf for sycl::half type [7c6541dc18e324e36853b473a15e8df4edf97755]
+- Fix nextafter with half on host [1f10f3588e8ccab79e42eee7fd640153d701b08c]
+- Fix identity values for (sub)group collectives min/max [2483893719324831eebce8c188aed688a8a04419]
+- Improve handling of large reqd_work_group_size values [25c3666dffa935c64db513d0db25205e0400c6b7]
+- Fix hip prefetch/mem_advise (noop) for rocm < 5.0 [7e4cc11139871da9e4662e535d4335b823b2118e]
+- Fix ranges to allow default constructor to 0s [0289d2ac94f9692e2067a3c32fb290169d84cbee]
+- Fix propagation of aspects from global variables [fb368a13044897d8ac693f72c442fe1f257c05d2]
+- Missing return statement in the multi_ptr<void..> conversion operator [836469ba88e292ce8dd662074117881c3fdd10d7]
+- Fix `convert` to `sycl::vec<bool, N>` [9dfaf274e2cbe42e992be0519fa7d2b0eb0face5]
+- Ensure queue_impl is not unnecessarily referenced in kernel fusion [a33b0a950ae679423aff505fdbd4a2cae77241eb]
+- Fix int overflow for OffsetInBytes in accessors [2fb0b140ad3be544552a1308fe49ce491bd2881a]
+- Fix invalid discard_write in atomic_memory_order_seq_cst [f69dae7972062d90f804c65996d31f246d2288b3]
+- Fix std::enable_if typo for marray and multi_ptr [0ef25ec3bdf8497dbe272e0295708d704cbe7020]
+- Fix integer type overflow in SYCLDeviceRequirements [cee07d3d26520fcdaab33dfaf7759b59210675d4]
+- Fix UB in group sort scratch memory access [ed4c01301a7c2be9157df6410be3b72ac4b95df7]
+- Use primary context in HIP adapter [d1c92cb95b29b0bf44683cc84b9978d767e0a300]
+- Allow passing default constructed accessors to kernels [fb8ddbf7b0ea3f22b6a2d7fce661e44123ebb2d5]
+- Fix issues with sycl_ext_oneapi_graph subgraphs [92ddf8d562d736f6c556976dce2c099e6aef591e]
+- Fix ambiguous log2 call in host code
+- Add device copyable trait for annotated_arg [9f65e61a2dbf9e9b631159a5ac2ec92137996f62]
```

Comment:
Looks more like an improvement because it was accompanied by a spec change.

I also think that saying something like "Made `annotated_arg<T>` implicitly device copyable if `T` is device copyable". Tagging @yug-intel

Reply:
Moved it to the improvements section